src/Illuminate/Database/Query/Builder.php

```diff
@@ -1299,6 +1300,169 @@ public function orWhereNotNull($column)
         return $this->whereNotNull($column, 'or');
     }
 
+    /**
+     * Add a where clause to determine if "date" column is in the past to the query.
+     *
+     * @param  string|array  $columns
+     * @param  string  $boolean
+     * @param  bool  $not
+     * @return $this
+     */
+    public function wherePast($columns, $boolean = 'and', $not = false)
+    {
+        $type = 'Basic';
+        $operator = $not ? '>=' : '<';
+        $value = Carbon::now()->format('Y-m-d H:i:s');
```

Comment:
Do you need the format(...?

Reply:
No, not for a full date. But I like the explicitness. Plus, it is needed for `whereToday`.